metricbeat/magefile.go

```diff
@@ -96,6 +98,27 @@ func Fields() error {
 	return mage.GenerateFieldsYAML("module")
 }
 
+func ExportDashboard() error {
```

Comment:
When you are done, document all the env vars that influence the behavior so that a user can run `mage -h exportDashboards` to see this information.